config/migrations/2022/20221004110503-Reglementaire-Bijlage-Type.ttl

```diff
@@ -0,0 +1,9 @@
+@prefix dossier: <https://data.vlaanderen.be/ns/dossier#> .
+@prefix gn: <http://data.lblod.info/vocabularies/gelinktnotuleren/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+gn:ReglementaireBijlage a owl:Class ;
```

Comment:
also add rdfs:Class

Reply:
Isn't owl:Class a subclass of rdfs:Class?